apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/ProjectsResourceTest.java

```diff
@@ -110,6 +118,7 @@ class ProjectsResourceTest {
     @BeforeAll
     void setUpAll(ClientSupport client, Jdbi jdbi) throws SQLException {
 
+        jdbi.setSqlLogger(new Slf4JSqlLogger());
```

Comment:
This also will print all sqls to the logs. I believe we only need it for debugging, so we can just add it in local

Reply:
Removed